src/extension/src/local_loggers/Logger.py

```diff
@@ -89,22 +76,31 @@ def log_debug(self, message):
     def log_verbose(self, message):
         """log verbose"""
         message = self.__remove_substring_from_message(message, Constants.ERROR_ADDED_TO_STATUS)
-        if self.telemetry_writer is not None and self.telemetry_writer.events_folder_path is not None:
-            self.telemetry_writer.write_event(self.telemetry_task, message, Constants.TelemetryEventLevel.Verbose)
-            message = self.__add_telemetry_error_note(message)
         if self.file_logger is not None:
             self.file_logger.write("\n" + self.VERBOSE + " " + "\n\t".join(message.strip().splitlines()).strip())
 
+    def log_telemetry_error(self, message):
+        """Log telemetry error"""
+        message = self.__remove_substring_from_message(message, Constants.ERROR_ADDED_TO_STATUS)
+        message = (self.NEWLINE_REPLACE_CHAR.join(message.split(os.linesep))).strip()
+        if self.file_logger is not None:
+            self.file_logger.write("\n" + self.TELEMETRY_ERROR + " " + message)
+        else:
+            print(self.TELEMETRY_ERROR + " " + message)
+
+    def log_telemetry(self, message):
+        """Log telemetry"""
```

Comment:
Note here in comment that this is exclusive for the telemetry class to call. To signify specialness (without taking it quite as far as private), consider naming the function _log_telemetry_module_error. Similarly for the other one.